content/learn/quick-start/getting-started/setup.md

```diff
@@ -130,17 +130,34 @@ It's not uncommon for debug builds using the default configuration to take multi
 Fortunately, there is a simple fix, and we don't have to give up our fast iterative compiles! Add the following to your `Cargo.toml`:
 
 ```toml
-# Enable a small amount of optimization in debug mode
+# Enable a small amount of optimization in debug mode.
 [profile.dev]
 opt-level = 1
 
-# Enable high optimizations for dependencies (incl. Bevy), but not for our code:
+# Enable a large amount of optimization in debug mode for dependencies.
 [profile.dev.package."*"]
 opt-level = 3
 ```
 
 You might think to simply develop in release mode instead, but we recommend against this as it can worsen the development experience by slowing down recompiles and disabling helpful debug symbols and assertions.
 
+In fact, you may want to optimize your release builds even further by adding the following to your `Cargo.toml`:
+
+```toml
+# Enable additional optimization in release mode at the cost of compile time.
+[profile.release]
+lto = "thin"
+codegen-units = 1
+
+# Optimize for size in wasm-release mode.
+[profile.wasm-release]
+inherits = "release"
+opt-level = "z"
+strip = "debuginfo"
```

Comment:
```suggestion
# Use release profile as default values
inherits = "release"

# Optimize with size in mind, also try "s", sometimes it is better.
# This doesn't increase compilation times compared to -O3, great improvements
opt-level = "z"

# Do a second optimization pass removing duplicate or unused code from dependencies.
# Slows compile times, marginal improvements
lto = "fat"

# When building crates, optimize larger chunks at a time
# Slows compile times, marginal improvements
codegen-units = 1

# String all debugging information from the binary
strip = "debuginfo"
```
From https://github.com/bevyengine/bevy/tree/main/examples#optimizing

Reply:
`codegen-units = 1` is implied by `inherits = "release"`, and I don't think `lto = "fat"` should actually be preferred over `lto = "thin"` tbh.

Reply:
You're right for the `codegen-units`, you're right. For the LTO I have no clue, but I want the two sources to at least match. I suppose we can remove the part in [bevyengine/bevy@main/examples#optimizing](https://github.com/bevyengine/bevy/tree/main/examples?rgh-link-date=2024-07-05T21%3A06%3A32Z#optimizing) and just link the website instead?

Reply:
Update: Added some of these comments. Also improved some of the comments here. Agreed that the two sources should match, though.

Reply:
Gonna mark this suggestion resolved. The other source should be updated, which will need to be a separate PR because it lives in another repo.